### PR TITLE
(feat): DP-6340 extend search messages

### PR DESCRIPTION
**Pull request by Zheltysheff**

### CODE REVIEW

File: proto/search.proto [hunk 1]
```diff
@@ -36,16 +36,24 @@ enum SearchContentType {
...
 }
 
 message SimpleMessageSearchCondition {
-    Peer peer = 1; /// where to search
-    string text = 2; /// search term
+    Peer peer = 1; // where to search
+    string text = 2; // search term
     oneof content_type {
-        SearchContentType type = 3; /// content message type to search
+        SearchContentType type = 3; // content message type to search
```
Comment (ostronom) [inline proto/search.proto:53]: oneof? :)

File: proto/search.proto [hunk 2]
```diff
@@ -36,16 +36,24 @@ enum SearchContentType {
...
     }
+    SearchDirection search_direction = 4; // search direction gte/lte
+    google.protobuf.Int64Value date = 5; // search message from
```
Comment (ostronom) [inline proto/search.proto:56]: date_from gives more insight w/o looking into comments (which are unavailable in IDE)

Comment (ostronom) [reply proto/search.proto:56]: And google.protobuf.Timestamp is better suited here

File: proto/search.proto (hunk 1 above)
Comment (Zheltysheff) [reply proto/search.proto:53]: fixed

File: proto/search.proto (hunk 2 above)
Comment (Zheltysheff) [reply proto/search.proto:56]: fixed

**Pull request merged**